features/mgmt_api/mgmt_api_topics.feature

```diff
@@ -0,0 +1,160 @@
+    Background: Setup companies and users
+        When we post to this "/companies"
+        """
+        [{"_id":"619277ef8bbbbfac6034aab4","name": "zzz company"}]
+        """
+        Then we get response code 201
+        When we post to this "/users"
+        """
+        {   
+            "_id":"619277ef8bbbbfac6034aab7",
+            "first_name": "Foo",
+            "last_name": "Bar",
+            "email": "foo@bar.org",
+            "company": "#companies._id#",
+            "user_type": "public"
+        }
+        """
+        Then we get response code 201
+
+    Scenario: Create a topic
+        When we post to this "/topics"
+        """
+        {
+            "label": "topic1",
+            "company": "#companies._id#",
+            "topic_type": "wire",
+            "query": "topic1",
+            "is_global": true,
+            "user": "#users._id#",
+            "subscribers": [
+                {"user_id": "#users._id#"}
+            ]
+        }
+        """
+        Then we get response code 201
+        When we get "/topics"
+        Then we get existing resource
+        """
+        {
+        "_items" :
+            [
+                {
+                    "label": "topic1",
+                    "company": "#companies._id#",
+                    "topic_type": "wire",
+                    "query": "topic1",
+                    "is_global": true,
+                    "user": "#users._id#",
+                    "subscribers": [
+                        {"user_id": "#users._id#", "notification_type": "real-time"}
+                    ]
+                }
+            ]
+        }
+        """
+
+    Scenario: Delete a topic
+        When we post to this "/topics"
+        """
+        [{
+            "label": "topic1",
+            "company": "#companies._id#",
+            "topic_type": "wire",
+            "query": "topic1",
+            "is_global": true,
+            "user": "#users._id#"
+        }]
+        """
+        When we delete latest
+        Then we get response code 204
+
+    Scenario: Update a topic
+        When we post to this "/topics"
+        """
+        [{
+            "label": "topic1",
+            "company": "#companies._id#",
+            "topic_type": "wire",
+            "query": "topic1",
+            "is_global": true,
+            "user": "#users._id#"
+        }]
+        """
+        When we patch latest
+        """
+        {"label": "topic2"}
+        """
+        Then we get updated response
+        """
+        {"label": "topic2"}
+        """
+
+    Scenario: Topic with folder
+        Given newsroom "topic_folders"
+        """
+        [
+            {   
+                "_id":"619277ef8bbbbfac6034aac1",
+                "name": "test",
+                "section": "wire"
+            }
+        ]
+        """
+        Given newsroom "topics"
+        """
+        [
+            {
+                "label": "topic1",
+                "company": "619277ef8bbbbfac6034aab4",
```

Comment:
Here you should be using `"#companies._id#"`, `"#users._id#"` and `"#topic_folders._id#"` for the IDs